ref.pl

```diff
@@ -349,6 +349,15 @@
     have an unselected radio button in that column.  To activate an
     inactive group, click its radio button.
 
+<p>
+    In order to reduce distraction when sketching in 2D, solid models from
+    inactive groups are "dimmed" so that they blend into the background.  To
+    disable this uncheck &rarr; Darken Inactive Solids user.  The intensity of
```

Comment:
```suggestion
    disable this uncheck &rarr; Darken Inactive Solids user.  
    By modifying the #def-dim-solid style any color may be assigned to geometry
    from inactive groups instead of just "dimming" them.
```